developer-docs-site/docs/reference/move.md

```diff
@@ -0,0 +1,110 @@
+
+## What is Move?
+
+Move is a safe and secure programming language for Web3 that emphasizes scarcity and access control. An assets in Move can be represented by or stored within *resource*. **Scarcity** is enforced by default, as unless otherwise made explicit objects and instances within Move are not copyable. Only those objects that have explicitly been defined at the bytecode layer as *copy* can be duplicated.
+
+Access control comes from both the notion of accounts as well as module access privileges. A module in Move may either be a library or a program that can be executed to create, store, or transfer assets. Move ensures that only public interfaces within a module may be accessed by other modules. In addition, structs or objects within Move can only be constructed within the module where it is defined unless the developer has explicitly defined a public constructor. Similarly, fields within a struct can only be accessed and mutated within the module that defines the struct or via public accessors and setters.
+
+In Move, a transaction's sender is represented by a `signer` or a verified owner of a specific account. The *signer* has the highest level of permission in Move and is the only entity capable of adding resources into an account. In addition, a module developer can require that a signer be present to access resources or modify assets stored within an account.
+
+## Comparison to other VMs
+
+| | Aptos / Move | Solana / SeaLevel | EVM |
+|---|---|---|---|
+| Data storage | Stored within the owner's account | Stored within the owner's account associated with a program | Stored within the account associated with a smart contract |
+| Parallelization | Capable of inferring parallelization at runtime within Aptos | Requires specifying within the transaction all accounts and programs accessed | Currently serial nothing in production | 
+| Transaction safety | Sequence number | Transaction uniqueness + remembering transactions | nonces, similar to sequence numbers |
+| Type safety | Module structs and generics | Program structs | Contract types |
+| Function calling | Explicit static dispatch | Static dispatch | Dynamic dispatch |
+
+## Aptos Move Features
+
+Each deployment of the MoveVM has the ability to extend the core MoveVM with additional features via an adapter layer. Furthermore, Move functions like a computer in which one can install a framework, which is similar to an operating system of a computer.
```

Comment:
Drop `of a computer` at the end, since you already said `like a computer` earlier.

Furthermore, Move(VM?) functions like a computer, where one can install a framework, which is similar to an operating system.